docs/standard/choosing-core-framework-server.md

```diff
@@ -19,94 +19,85 @@ Use .NET Core for your server application when:
-
-### .NET technologies not available for .NET Core
-
-Some .NET Framework technologies aren't available in .NET Core. Some of them might be available in later .NET Core releases. Others don’t apply to the new application patterns targeted by .NET Core and may never be available. The following list shows the most common technologies not found in .NET Core:
-
-- ASP.NET Web Forms applications: ASP.NET Web Forms are only available in .NET Framework. ASP.NET Core cannot be used for ASP.NET Web Forms. There are no plans to bring ASP.NET Web Forms to .NET Core.
+- ASP.NET Web Forms applications: ASP.NET Web Forms are only available in .NET Framework. ASP.NET Core cannot be used for ASP.NET Web Forms.
 
 - ASP.NET Web Pages applications: ASP.NET Web Pages aren't included in ASP.NET Core.
 
-- WCF services implementation. Even when there’s a [WCF client library](https://github.com/dotnet/wcf) to consume WCF services from .NET Core, WCF server implementation is currently only available in .NET Framework. This scenario is not part of the current plan for .NET Core, but it’s being considered for the future.
+- WCF services implementation. Even when there’s a [WCF client library](https://github.com/dotnet/wcf) to consume WCF services from .NET 5, WCF server implementation is currently only available in .NET Framework.
 
-- Workflow-related services: Windows Workflow Foundation (WF), Workflow Services (WCF + WF in a single service), and WCF Data Services (formerly known as "ADO.NET Data Services") are only available in .NET Framework. There are no plans to bring these technologies to .NET Core.
+- Workflow-related services: Windows Workflow Foundation (WF), Workflow Services (WCF + WF in a single service), and WCF Data Services (formerly known as "ADO.NET Data Services") are only available in .NET Framework.
 
-- Language support: Visual Basic and F# are currently supported in .NET Core, but not for all project types. For a list of supported project templates, see [Template options for dotnet new](../core/tools/dotnet-new.md#arguments).
+- Language support: Visual Basic and F# are currently supported in .NET 5, but not for all project types. For a list of supported project templates, see [Template options for dotnet new](../core/tools/dotnet-new.md#arguments).
 
-### Platform doesn’t support .NET Core
+### Platform doesn’t support .NET 5
 
-Some Microsoft or third-party platforms don’t support .NET Core. Some Azure services provide an SDK not yet available for consumption on .NET Core. This is a transitional circumstance, as all of Azure services use .NET Core. In the meantime, you can use the equivalent REST API instead of the client SDK.
+Some Microsoft or third-party platforms don’t support .NET 5. Some Azure services provide an SDK not yet available for consumption on .NET 5. In such cases, you can use the equivalent REST API instead of the client SDK.
```

Comment:
```suggestion
Some Microsoft or third-party platforms don't support .NET 5. Some Azure services provide an SDK not yet available for consumption on .NET 5. In such cases, you can use the equivalent REST API instead of the client SDK.
```

You must have the automatic replacement feature for apostrophes turned off in VS Code.

Reply:
I didn't know about that feature - I'll look it up, thanks.

Reply:
https://review.docs.microsoft.com/en-us/help/contribute/docs-authoring/smart-quote-replacement?branch=master

Reply:
Thanks for the review and the VS Code info!